sycl/include/CL/__spirv/spirv_types.hpp

```diff
@@ -145,6 +145,8 @@ struct ConstantPipeStorage {
   int32_t _Capacity;
 };
 
+struct __spirv_TaskSequenceINTEL;
+
```

Comment:
This should be inside the __spv namespace.   Suggest adding it after the __spirvJointMatrixINTEL type definition above.